src/modules/content-seeding/news-aggregator.js

```diff
@@ -0,0 +1,290 @@
+/* flow */
+/* eslint-disable no-console, consistent-this, no-cond-assign */
+
+import feedParser from 'feed-read';
+import _ from 'lodash';
```

Comment:
instead of importing whole lodash, you should import individual modules instead, e.g. - 

```js
import map from 'lodash/map';
import pick from 'lodash/pick';
import orderBy from 'lodash/orderBy';
import uniqueBy from 'lodash/uniqueBy';
```

A good practice to follow, even if this doesn't have much impact in server, in client it's important to include only what you need.